client/jetpack-connect/plans-grid.jsx

```diff
@@ -4,11 +4,12 @@
  */
 import React, { Component } from 'react';
 import PropTypes from 'prop-types';
-import { localize } from 'i18n-calypso';
+import { localize, moment } from 'i18n-calypso';
```

Comment:
No need to import `moment`, it's attached with `localize`.

Reply:
oh! TIL!